resources/icecube_setup.py

```diff
@@ -1,207 +1,388 @@
-import os
-import sys
 import argparse
 import asyncio
+import os
+import sys
+from typing import Dict, TypedDict
 
-import requests
-
-sys.path.append(os.getcwd())
-
-from krs.bootstrap import bootstrap, user_mgmt_app, get_token
+# local imports
+from krs.bootstrap import bootstrap, get_token, user_mgmt_app
 from krs.groups import create_group
 from krs.token import get_rest_client
 
+sys.path.append(os.getcwd())
+
 
 GROUPS = {
-    'institutions': {
-        'IceCube': {},
-        'IceCube-Gen2': {},
-        'HAWC': {},
-        'CTA': {},
-        'ARA': {},
-    },
-    'experiments': {
-        'IceCube': {
-            'Working Groups': {},
-        },
-    },
-    'posix': {},
-    'tokens': {},
+    "institutions": {
+        "IceCube": {},
+        "IceCube-Gen2": {},
+        "HAWC": {},
+        "CTA": {},
+        "ARA": {},
+    },
+    "experiments": {"IceCube": {"Working Groups": {},},},
+    "posix": {},
+    "tokens": {},
```

Comment:
Is there a reason to switch the quote type?

Reply:
not at all. I use black to auto format, so I must've missed this chunk